### DISCUSSION TITLE
Setting up a build server

### DISCUSSION THREAD

**Discussion by hamishmb** [Homebrew/discussions#1158] [Tap maintenance and Homebrew development] [answered]
Hi there,

I have noticed that you don't hav many pre-compiled packages for macOS Catalina and Big Sur.

I am happy to set up a spare Linode instance I have as a build server, and a repository to help with this. It will be slow, but is idle most of the time and has enough RAM, so it should be pretty effective once it catches up.

Is there a guide or any help you can give me to do this, and are you interested? The other concern I have is how much storage space will be required for this. Can anyone give me an idea of the scale?

**Comment by SMillerDev** [comment, marked as answer]
> I have noticed that you don't hav many pre-compiled packages for macOS Catalina and Big Sur.

I'm not sure where you spotted this but almost everything is precompiled for those macOS versions including ARM based packages for Big Sur.

> I am happy to set up a spare Linode instance I have as a build server, and a repository to help with this. It will be slow, but is idle most of the time and has enough RAM, so it should be pretty effective once it catches up.

We only manage our own build servers to avoid malicious interference with the build process. Thanks for the offer though.

**Reply by hamishmb** [reply]
Must have been old news, my bad.

And no worries, I totally understand the need for control :)
